README.md

```diff
@@ -19,14 +19,19 @@ import { I18n, I18nFlavor } from "https://deno.land/x/grammy_i18n/mod.ts";
 
 ## Example
 
-Example project structure:
+An example project structure, you can also seperate your translations into different files to make it easier to maintain large projects.
+Nested translations don't change how you use translation keys, so nothing should break if you decide to use them.
```

Comment:
Can this be a bit more clear?
The file location or name is irrelevant? Only the content of the files is relevant?

Reply:
Not sure if I fully understand your comment, but would something like this clear things up?

```suggestion
By splitting translations you don't change how you retrieve the keys contained within them, so for example, a key called `greeting` which is located in both the `locales/it.ftl` and `locales/en/dialogues/greeting.ftl` files can be retrieved by simply using `ctx.t("greeting")`.
```

Reply:
Yes, I like this suggestion

Reply:
I would switch to „Located in either …“. Then the key is not duplicated somewhere which might create another confusion. Otherwise I like the new phrasing a lot better as it is way more clear what to expect.

Reply:
Good point, I've fixed this in the latest commit, feel free to resolve if you think everything is clear enough.